<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secDialog</class>
 <widget class="QDialog" name="secDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #555555;</string>
  </property>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#connect {
	border-radius: 15px;
	border-style: solid;
	border-width:1px;
	border-color:black;
	padding: 2px;
	background-color: #F2DCBB;
	font: bold 18px;
	color: charcoal;
}
QPushButton#connect:hover {
	background-color: #CC9966;
}
QPushButton#connect:disabled {
	background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="surrender">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#surrender {
	border-radius: 15px;
	border-style: solid;
	border-width:1px;
	border-color:black;
	padding: 2px;
	background-color: #F2DCBB;
	font: bold 17px;
	color: charcoal;
}
QPushButton#surrender:hover {
	background-color: #CC9966;
}
QPushButton#surrender:disabled {
	background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>Surrender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#exit {
	border-radius: 15px;
	border-style: solid;
	border-width:1px;
	border-color:black;
	padding: 2px;
	background-color: #F2DCBB;
	font: bold 18px;
	color: charcoal;
}
QPushButton#exit:hover {
	background-color: #CC9966;
}
QPushButton#exit:disabled {
	background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="explain">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>221</y>
     <width>301</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: white;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Serif CJK KR','Chilanka'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:15px; color:#ff0000;&quot;&gt;&amp;lt;How to Play&amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;1. 호스트의 ip주소를 입력 후 &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;connect&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;를 누른다. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;2. 연결 후, 준비가 되면 &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;start&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt; 눌러 게임을 시작.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;3. 놓을 곳을 누르고 put 버튼을 누르고 차례를 넘긴다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;4. 자신의 차례일 때 항복하려면 &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;surrender&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;를 누른다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;5. &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;exit&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;를 눌러서 프로그램을 종료한다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:15px; color:#ff0000;&quot;&gt;&amp;lt;Game Rule&amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;&amp;gt; 호스트가 &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;선공(O)&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;, 게스트가 &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;후공(X)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;번갈아&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt; 가며 O 와 X를 3x3 판에 놓는다.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;&amp;gt; 자신의 말이 &lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:600;&quot;&gt;가로 or 세로 or 대각선&lt;/span&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;이 같으면 승리&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="debugging">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>301</width>
     <height>115</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
border-style: solid;
border-width:3px;
border-color:white;
background-color: gray;
color:white;
font-style:bold;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Chilanka'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;Hello Tic Tac Toe!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="board_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_1 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
font: 50px;
background-color: #F9F7CF;
}
QPushButton#board_1:hover {
background-color: #CCCC99;
}
QPushButton#board_1:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_2 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
font: 50px;
background-color: #F9F7CF;
}
QPushButton#board_2:hover {
background-color: #CCCC99;
}
QPushButton#board_2:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_3 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_3:hover {
background-color: #CCCC99;
}
QPushButton#board_3:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_4 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_4:hover {
background-color: #CCCC99;
}
QPushButton#board_4:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_5 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_5:hover {
background-color: #CCCC99;
}
QPushButton#board_5:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_6 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_6:hover {
background-color: #CCCC99;
}
QPushButton#board_6:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_7{
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_7:hover {
background-color: #CCCC99;
}
QPushButton#board_7:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_8 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_8:hover {
background-color: #CCCC99;
}
QPushButton#board_8:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#board_9 {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: #F9F7CF;
font:50px;
}
QPushButton#board_9:hover {
background-color: #CCCC99;
}
QPushButton#board_9:disabled {
background-color: lightgray;
font:50px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="opscore">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>90</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#opscore {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: white;
}
QLineEdit#opscore:disabled {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: white;
color:black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="myscore">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>90</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#myscore {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: white;
}
QLineEdit#myscore:disabled {
border-radius: 5px;
border-style: solid;
border-width:1px;
border-color:black;
background-color: white;
color:black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#start {
	border-radius: 15px;
	border-style: solid;
	border-width:1px;
	border-color:black;
	padding: 2px;
	background-color: #F2DCBB;
	font: bold 17px;
	color: charcoal;
}

QPushButton#start:hover {
	background-color: #CC9966;
}

QPushButton#start:disabled {
	background-color: gray;
}
</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="put">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chilanka</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#put {
	border-radius: 15px;
	border-style: solid;
	border-width:1px;
	border-color:black;
	padding: 2px;
	background-color: orange;
	font: bold 20px;
	color: white;
}

QPushButton#put:hover {
	background-color: darkorange;
}

QPushButton#put:disabled {
	background-color: gray;
}</string>
   </property>
   <property name="text">
    <string>Put</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>93</y>
     <width>7</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #BBBBBB;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>85</y>
     <width>201</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #BBBBBB;
border-radius: 5px;
border-style: solid;
border-width: 1px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chilanka</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BBBBBB;
border-style: none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:x-large;&quot;&gt;Score &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>15</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #BBBBBB;
border-style: solid;
border-radius: 5px;
border-width: 1px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>15</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #BBBBBB;
border-style: solid;
border-radius: 5px;
border-width: 1px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>connect</zorder>
  <zorder>ip</zorder>
  <zorder>surrender</zorder>
  <zorder>exit</zorder>
  <zorder>explain</zorder>
  <zorder>debugging</zorder>
  <zorder>board_1</zorder>
  <zorder>board_2</zorder>
  <zorder>board_3</zorder>
  <zorder>board_4</zorder>
  <zorder>board_5</zorder>
  <zorder>board_6</zorder>
  <zorder>board_7</zorder>
  <zorder>board_8</zorder>
  <zorder>board_9</zorder>
  <zorder>opscore</zorder>
  <zorder>myscore</zorder>
  <zorder>start</zorder>
  <zorder>put</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
